docs/UsersGuide.asciidoc

```diff
@@ -1435,11 +1435,28 @@ Further remarks:
   section of <<GettingStarted.asciidoc#_configuration,the web UI configuration>>.
   Jobs outside of any group use the limits configured in the `[no_group_limits]`
   section of the web UI configuration.
-* Archiving of important jobs can be enabled. Checkout the related settings
-  within the `[archiving]` section of the config file for details.
 * The <<Installing.asciidoc#auditing,Auditing>> section explains the cleanup of
   the audit log.
 
+[id="archiving"]
+=== Automatic archiving of old jobs ===
+
+Archiving of important jobs can be enabled:
+[source,ini]
+--------------------------------------------------------------------------------
+[archiving]
+## Moves results of important jobs which are preserved during the cleanup
+## to OPENQA_ARCHIVEDIR (/var/lib/openqa/archive/ by default)
+#archive_preserved_important_jobs = 0
+--------------------------------------------------------------------------------
+
+Currently the primary use case is keeping important jobs in a dedicated
+directory once they expire. Results of such jobs are periodically moved to
+`OPENQA_ARCHIVEDIR`. In the web UI an icon indicates that a job was archived.
+
+Note that retention periods are the same as for important logs and archiving
+is not reversible.
```

Comment:
The above comment in line 1448 reads nicely but here it's a bit convoluted. The first sentence says "keeping important jobs […] once they expire" which is contradicting. And results are not really moved periodically but when the according cleanup is triggered. How about the following to make it clear that the alternatives are deletion or archiving:

```suggestion
When archiving is enabled then job results that reach their expiration but are
considered important are moved to the archive. In the web UI an icon
indicates that a job was archived. As only important jobs are considered
for archiving the retention periods for important job results and logs also
apply here.
```